packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+numbers. A Service segregates data for a specific use case or campaign. Users can set up one or more services
+within a given product. Services are currently used in Messaging, Chat, Verify, Authy, Functions, Studio,
+SIP trunking, Voice BYOC, TaskRouter, Conversations, Notify, Proxy, Autopilot, Assets, and Sync.
+If you have services in your product, they fall under the "Manage" category in the "Develop" tab.
+
+### Service navigation guidelines
+
+A Service navigation pattern should only be used for actual services. They're designed to provide a focused view
+on just the pages relevant to that particular service. You get to a service from a list of services
+in your product nav under Develop.
+
+![](../../../assets/images/experiences/console/service-list-service-detail.png)
+
+<ImageCaption>
+  When you choose a service from a list, it opens up a page focused on that service's details.
+</ImageCaption>
+
+---
+
+## Account-level navigation
+
+With Console’s focus on jobs-to-be-done, Account-level menus are another important area. Account-level sections
+like "Account", "Billing", and "Support" provide designated areas to perform account administration separate
+from the products you may use in that account.
+
+**Primary personas:** Billing managers, IT Admins, Customer support
+
+Jobs to be done: Manage billing, add users to an account and assign roles, manage compliance
+
+We currently have 3 account-level areas:
+
+1. **Account:** Accessed from the masthead.
+2. **Billing:** Accessed from the masthead.
+3. **Support Center:** Accessed from Docs &amp; support menu in left nav.
+
+If there is a need to access a page in an account-level area from your product, such as API Keys or Credentials,
+link to the page from your product rather than duplicating the page inside of your product.
+This eliminates the need to duplicate content in multiple places.
+
+## Vertical sidebar navigation vs in-page horizontal tabs
+
+The left sidebar navigation should include all the highest-level primitives related to your product.
+Within a given page, horizontal tabs can be used to segment different categories of information related to
+that primitive, or different variations of that primitive.
+
+## When to use breadcrumbs
+
+Breadcrumbs are used sparingly. A breadcrumb is only visible when viewing a page that is secondary or tertiary
+to the page selected in the main navigation. The breadcrumb gives you the ability to link back to the primary page.
+It also helps to remind you where you are in the hierarchy of pages.
+
+If you were to navigate to a tertiary page, the breadcrumb would show both the primary and secondary breadcrumbs
+on the tertiary page ( Logs / Log detail / ). The first breadcrumb will always match the page
+highlighted in the nav.
+
+![](../../../assets/images/experiences/console/breadcrumb-example.png)
+
+<ImageCaption>A screenshot of breadcrumbs being used on a Console page</ImageCaption>
```

Comment:
Suggestion(non-blocking): could this be more descriptive of what the image is? How is this breadcrumb being used sparingly? Is the fact that its at the top of the page important?

Reply:
Agreed that these need some work. I could use some @twilio-labs/design-systems-pd help on these captions!